public/scripts/textgen-settings.js

```diff
@@ -116,6 +116,10 @@ const settings = {
     dynatemp_exponent: 1.0,
     smoothing_factor: 0.0,
     smoothing_curve: 1.0,
+	dry_allowed_length: 2,				   
+	dry_multiplier: 0.0,				 
+	dry_base: 1.75,			
+	dry_sequence_breakers: '',
```

Comment:
```suggestion
	dry_sequence_breakers: '[]',
```

An empty string is not valid JSON. Note that the syntax of this parameter is currently being discussed upstream.